JoinColony/colonyCDapp
# Fix: incorrect latest colony version in the DB

## Description

My best guess as to what happened is that after seeding the DB with the correct latest version (`15`), ingestor picked up two `ColonyVersionAdded` "adding" versions 3 and 4. Because there was no mechanism ensuring we only store the highest version we've seen so far, the version was being incorrectly set to 4.

The fix is mostly on block-ingestor side: https://github.com/JoinColony/block-ingestor/pull/197

## Testing

1. Start the dev environment. Wait until all containers are running.
2. Run the following query:
```gql
query ListVersions {
  listCurrentVersions {
    items {
      key
      version
    }
  }
}
```
Previously, entry with key `colony` had a version of `4` which should've now been changed to `15`.

As I came across this when trying to create a new colony, it might be worth running that flow as well.

## Diffs

**Deletions** ⚰️

* `setCurrentVersion` lambda

Resolves #2088
